source/javascripts/gradient_fade.js

```diff
@@ -0,0 +1,16 @@
+function init() {
+    // Create the listener function
+    var updateLayout = _.debounce(function(e) {
```

Comment:
The header there is using the `transform3d` hack https://github.com/artsy/folio.artsy.net/blob/484d6049aa30c01b6d3b87072b1cafb32da79294/source/stylesheets/_header.sass#L13 FWIW